next/@types/network.ts

```diff
@@ -107,10 +102,8 @@ export type IntegratedAtlasRow = AtlasRow &
   Pick<IntegratedAtlas, "cxgId" | "cxgURL" | "datasetAssets" | "name">;
 
 export interface DatasetAsset {
-  assetId: string;
-  datasetId: string;
   downloadURL: string;
-  fileName: string;
+  fileSize: number;
```

Comment:
`filesize` is returned from CELLxGENE dataset API - we can now display the download file size.